Documentation/botr/mixed-mode.md

```diff
@@ -0,0 +1,36 @@
+# Mixed Mode Assemblies
+## Introduction
+Most interoperability between managed and native code uses P/Invokes, COM, or WinRT. Since P/Invokes and COM are bound to native code at runtime, they’re susceptible to mistakes ranging from incorrect naming to subtle mistakes in signatures that cause stack corruption. COM can also be used to call from native to managed code, but it requires a significant amount of setup. WinRT avoids those problems but isn’t available in all cases.
```

Comment:
>  COM are bound to native code at runtime, they’re susceptible to mistakes ranging from incorrect naming to subtle mistakes in signatures that cause stack corruption

In the typical workflow, COM interop is auto-generated using tlbexp/tlbimp tools that avoids the mistakes that lead to the stack corruption. A lot of folks using COM do not even know that it is possible to write the COM interfaces by hand in C#.

Reply:
Using Tlbexp/Tlbimp will avoid these incorrect naming or stack corruption using COM or exporting COM.

Reply:
Removed the COM mistakes mention